packages/phoenix-evals/src/phoenix/evals/span_templates.py

```diff
@@ -186,6 +186,86 @@
 
 QA_SPAN_PROMPT_RAILS_MAP = OrderedDict({True: "correct", False: "incorrect"})
 
+FUNCTION_CALLING_SPAN_BASE_TEMPLATE = """
+    You are an evaluation assistant evaluating questions and function calls to
+    determine whether the function called would answer the question. The function
+    calls have been generated by a separate agent, and chosen from the list of
+    functions provided below. It is your job to decide whether that agent chose
+    the right function to call. The agent can also choose not to call a function,
```

Comment:
I think the picture is more complex now since multiple tools/ functions can be called at once. This looks like it's the default behavior for tool calling.

https://platform.openai.com/docs/api-reference/chat/create#chat-create-parallel_tool_calls